src/containers/PipelineRuns/PipelineRuns.js

```diff
@@ -149,21 +200,53 @@ export /* istanbul ignore next */ class PipelineRuns extends Component {
             lowContrast
           />
         )}
-        <Button
-          className="create-pipelinerun-button"
-          iconDescription="Create PipelineRun"
-          renderIcon={Add}
-          onClick={() => this.toggleModal(true)}
-        >
-          Create PipelineRun
-        </Button>
-        <CreatePipelineRun
-          open={this.state.showCreatePipelineRunModal}
-          onClose={() => this.toggleModal(false)}
-          onSuccess={this.handleCreatePipelineRunSuccess}
-          pipelineRef={pipelineName}
-          namespace={selectedNamespace}
-        />
+        <div className="search-bar">
+          <Search
+            placeHolderText="Input a filter e.g. gitRepo:https://github.com/example/helloworld"
```

Comment:
This all looks great, the only thing I'd add is something in the UI that mentions it's by label only - it's unclear atm despite the above example that this is a key value pair and that you can't, for example, just search on `success`, or by a certain pipeline name. I'd prefer this to be done via the UI and not through docs - perhaps 'Add label filter' instead of 'Add filter', or making use of the placeholder text